lib/elements/prompt.js

```diff
@@ -47,6 +47,11 @@ class Prompt extends EventEmitter {
     this.in.on('keypress', keypress);
   }
 
+  ceos() {
+    readline.clearScreenDown(this.out);
+    return ``;
+  }
+
```

Comment:
Not sure if this is where this should be located.  I put it here because this is where `readline` is.  Just kind of a quick hack.

Reply:
`erase.down()` is a thing..